<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDashboard</class>
 <widget class="QMainWindow" name="mainDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainDashboard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="spendingChartPlaceholder" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>761</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="spendingsButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>700</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Spendings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incomeButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>700</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Income</string>
    </property>
   </widget>
   <widget class="QWidget" name="spendSaveChartPlaceholder" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>901</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="budgetLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Budget Left:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="budgetLabel_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Current Income:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="budgetLabel_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Current Savings:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="updateBudgetLeftLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="updateCurrentIncomeLeftLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="updateCurrentSavingsLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="emptyLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>It's looking a little empty. Click on the Spendings or Income buttons to see your cash flow!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>134</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="calculatorButton">
       <property name="text">
        <string>Calculator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateBudgetButton">
       <property name="text">
        <string>Update Budget</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionUser_Settings"/>
    <addaction name="actionLog_Out"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionUser_Settings">
   <property name="text">
    <string>User Settings</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
